policyengine_us/parameters/gov/irs/deductions/itemized/reduction/agi_threshold.yaml

```diff
@@ -1,56 +1,155 @@
+    2026-01-01: 339_850
+    2027-01-01: 347_050
+    2028-01-01: 354_050
+    2029-01-01: 361_050
+    2030-01-01: 368_200
+    2031-01-01: 375_500
+    2032-01-01: 382_900
+    2033-01-01: 390_500
+    2034-01-01: 398_250
+    2035-01-01: 406_200
+  metadata:
+    uprating:
+      parameter: gov.irs.uprating
+      rounding:
+        type: downwards
+        interval: 50
 JOINT:
-  2009-01-01: 166_800
-  2013-01-01: 150_000
-  2014-01-01: 152_525
-  2015-01-01: 154_950
-  2016-01-01: 155_650
-  2017-01-01: 156_900
+  values:
+    2009-01-01: 250_200
+    2010-01-01: 0
+    2013-01-01: 300_000
+    2014-01-01: 305_050
+    2015-01-01: 309_900
+    2016-01-01: 311_300
+    2017-01-01: 313_800
+    2018-01-01: 0
+    2026-01-01: 414_836
+  metadata:
+    uprating:
+      parameter: gov.irs.uprating
+      rounding:
+        type: downwards
+        interval: 50
+
 SEPARATE:
-  2009-01-01: 83_400
-  2013-01-01: 75_000
-  2014-01-01: 76_262.5
-  2015-01-01: 77_475
-  2016-01-01: 77_825
-  2017-01-01: 78_450
+  values:
+    2009-01-01: 125_100
+    2010-01-01: 0
+    2013-01-01: 150_000
+    2014-01-01: 152_525
+    2015-01-01: 154_950
+    2016-01-01: 155_650
+    2017-01-01: 156_900
+    2018-01-01: 0
+    # CBO forecast.
+    2026-01-01: 203_900
+    2027-01-01: 208_200
+    2028-01-01: 212_450
+    2029-01-01: 216_650
+    2030-01-01: 220_900
+    2031-01-01: 225_300
+    2032-01-01: 229_750
+    2033-01-01: 234_300
+    2034-01-01: 238_950
+    2035-01-01: 243_700
+  metadata:
+    uprating:
+      parameter: gov.irs.uprating
+      rounding:
+        type: downwards
+        interval: 50
+
 HEAD_OF_HOUSEHOLD:
-  2009-01-01: 166_800
-  2013-01-01: 150_000
-  2014-01-01: 152_525
-  2015-01-01: 154_950
-  2016-01-01: 155_650
-  2017-01-01: 156_900
+  values: 
+    2009-01-01: 208_500
+    2010-01-01: 0
+    2013-01-01: 275_000
+    2014-01-01: 279_650
+    2015-01-01: 284_050
+    2016-01-01: 285_350
+    2017-01-01: 287_650
+    2018-01-01: 0
+    # CBO forecast.
+    2026-01-01: 373_850
+    2027-01-01: 381_750
+    2028-01-01: 389_500
+    2029-01-01: 397_200
+    2030-01-01: 405_000
+    2031-01-01: 413_050
+    2032-01-01: 421_200
+    2033-01-01: 429_550
+    2034-01-01: 438_100
+    2035-01-01: 446_800
+  metadata:
+    uprating:
+      parameter: gov.irs.uprating
+      rounding:
+        type: downwards
+        interval: 50
+
 SURVIVING_SPOUSE:
-  2009-01-01: 166_800
-  2013-01-01: 150_000
-  2014-01-01: 152_525
-  2015-01-01: 154_950
-  2016-01-01: 155_650
-  2017-01-01: 156_900
+  values:
+    2010-01-01: 0
+    2013-01-01: 300_000
+    2014-01-01: 305_050
+    2015-01-01: 309_900
+    2016-01-01: 311_300
+    2017-01-01: 313_800
+    2018-01-01: 0
+    # CBO forecast.
+    2026-01-01: 407_850
+    2027-01-01: 416_450
+    2028-01-01: 424_900
+    2029-01-01: 433_300
+    2030-01-01: 441_850
+    2031-01-01: 450_600
+    2032-01-01: 459_500
+    2033-01-01: 468_600
+    2034-01-01: 477_900
+    2035-01-01: 487_400
+  metadata:
```

Comment:
can we use propagage_metadata_to_children since it's the same $50 rule?

Reply:
It should work, should I change it or should we keep the current uprating?

Reply:
use propagate